CHANGELOG.md

```diff
@@ -75,11 +75,32 @@ different versioning scheme, following the Haskell community's
   `const_reference` typedefs have been removed from `bond::nullable<T>`.
 * **Breaking change** The `Allocator` (second) type parameter has be removed from
   `bond::nullable<T>` and now it is always deduced from `T`.
-* **Breaking change** When using Bond-over-gRPC, the generated `ClientCore::Async*`
-  functions are now accepting the `std::shared_ptr<grpc::ClientContext>`
-  as the last parameter.
-* **Breaking change** When using Bond-over-gRPC, the client callback now directly accepts
-  `bond::ext::gRPC::unary_call_result<Response>` (drops the `std::shared_ptr`).
+* **Breaking change** The `bond::capped_allocator` and related types have been
+  moved to the `bond::ext` namespace and the "bond/ext" include directory.
+* **Breaking changes** in Bond-over-gRPC (based on real-world use and feedback):
+  - The generated `ClientCore` and `ServiceCore` class templates and the
+    `Client` and `Service` convenience typedefs have all been replaced with
+    normal classes named `Client` and `Service`. The `ThreadPool` type parameter
+    has been removed in favor of simplified runtime representation of a `Scheduler`.
+  - The `Scheduler` concept and the `bond::ext::gRPC::thread_pool` implementation
```

Comment:
We should mention something about services now needing to take a scheduler during construction.